test/unit/sites.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const chai = require('chai');
+const assert = chai.assert;
+const sites = require('../../routes/sites');
+
+describe('utils', function() {
+  describe('sites', function() {
+    it('gets all sites', function(done) {
+      var fakeData1 = {
+        site_id: 0,
+        site_name: 'fakeSiteName',
+        site_address: '123 Boston, MA'
+      };
+      var fakeData2 = {
+        site_id: 22222,
+        site_name: 'fakeSiteName',
+        site_address: '123 Boston, MA'
+      };
+      var promise = sites.getSites({});
+
+      promise.then(function(data) {
+        assert.deepEqual([fakeData1, fakeData2], data);
+        done();
+      });
+    });
+
+    it('gets all sites for a given coach', function(done) {
+      var fakeData1 = {
+        site_id: 1,
+        site_name: 'for the coach',
+        site_address: '123 Boston, MA'
+      };
+      var fakeData2 = {
+        site_id: 2,
+        site_name: 'also for the coach',
+        site_address: '123 Boston, MA'
+      };
+      var promise = sites.getSites({
+        req: {
+          coach_id: 1
```

Comment:
Wouldn't the coach_id: 1 be under query {}?
Like:
req: {
query: {
coach_id: 1
}
}

All query params are stored inside queries {}